docs/eos-learn/eos-network-foundation.md

```diff
@@ -0,0 +1,101 @@
+---
+id: eos-network-foundation
+title: Blue Papers EOS Network Foundation
+sidebar_label: Blue Papers
```

Comment:
should we include the meta description and keywords for these new pages (see #394 #424 )